pyproject.toml

```diff
@@ -5,3 +5,11 @@ directory = "CHANGES/"
 title_format = "{version} ({project_date})"
 template = "CHANGES/.TEMPLATE.rst"
 issue_format = "`#{issue} <https://pulp.plan.io/issues/{issue}>`_"
+
+    [[tool.towncrier.section]]
+        path = ""
+        name = ""
+
+    [[tool.towncrier.section]]
```

Comment:
I was trying to suggest on slack to use `tool.towncrier.type`. What I said on slack was confusing.

Reply:
Similar to the last part of this readme https://github.com/twisted/towncrier#further-options